### PR TITLE
more vars for static inventory

### LINKED ISSUE #406: No hosts file under /etc/ansible
I've installed Openshift Origin by executing `ose-on-aws.py`. The cluster is now operative but I'd like to install metrics and logging. According to the documentation I should follow the [Advanced Installation](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-ansible). I'm not able to find `/etc/ansible/hosts` file to configure Ansible. Any idea about that? My idea was to use `- include: /usr/share/ansible/openshift-ansible/playbooks/byo/config.yml` but what should I use as inventory? The EC2 hosts inventory or a plain file configured as wanted?

### LINKED ISSUE #408: Containerized not as parameter
I don't know If I'm wrong but `ose-on-aws.py` has a parameter `containerized`. This parameter is not parametrized in the `openshift-setup.yaml`. Is it the expected behaviour?

### PR DESCRIPTION
**Pull request by cooktheryan**
Resolves #406

### CODE REVIEW

File: reference-architecture/aws-ansible/playbooks/roles/inventory-file-creation/files/inventory [hunk 1]
```diff
@@ -24,11 +24,11 @@ openshift_master_cluster_public_hostname=openshift-master.{{ public_hosted_zone
 osm_default_subdomain={{ wildcard_zone }}
 openshift_master_default_subdomain={{ wildcard_zone }}
 osm_default_node_selector="role=app"
-deployment_type=openshift-enterprise
-os_sdn_network_plugin_name=redhat/{{ openshift_sdn | default('openshift-ovs-subnet') }}
+deployment_type={{ deployment_type | default('openshift-enterprise') }}
+os_sdn_network_plugin_name={{ openshift_sdn | default('redhat/openshift-ovs-subnet') }}
```
Comment (e-minguez) [inline reference-architecture/aws-ansible/playbooks/roles/inventory-file-creation/files/inventory]: I'd say to use the multitenant plugin by default for security reasons

Comment (eye0fra) [inline reference-architecture/aws-ansible/playbooks/roles/inventory-file-creation/files/inventory]: I agree

**Comment by glennswest** [comment]
Think it's safer to do it that way 
It was a intentional choice

**Comment by glennswest** [comment]
The other reason you uses this, is it should make mdns discovery work properly, otherwise mdns will go all over the place, and not what you want.

**Comment by cooktheryan** [comment]
@e-minguez you good with approving this one and a subsequent change of me switching the default SDN there. It would require some doc updates that I don't want to have to force in at the moment.

**Comment by e-minguez** [comment]
aos-ci-test

**Comment by e-minguez** [comment]
aos-ci-test

**Pull request merged**